db/schema.rb

```diff
@@ -54,6 +62,9 @@
     t.datetime "remember_created_at"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.boolean "admin", default: false
```

Comment:
Don't forget about functionally 
The first registered user should become and admin

Reply:
@VolodymyrZyblikevich